resources/tests.mjs

```diff
@@ -350,6 +350,7 @@ Suites.push({
     async prepare(page) {
         await page.waitForElement("#render");
     },
+    withReactAct: true,
```

Comment:
When this is set to true, then the runner will wrap the user actions in a call to an exposed `ReactAct` by the workload.